src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -66,7 +66,9 @@ ALWAYS_INLINE size_t resolveProgramCounter(char* codeBuffer, const size_t progra
 Value ByteCodeInterpreter::interpret(ExecutionState& state, ByteCodeBlock* byteCodeBlock, size_t programCounter, Value* registerFile, void* initAddressFiller)
 {
 #if defined(COMPILER_GCC)
-    *((size_t*)initAddressFiller) = ((size_t) && FillOpcodeTableOpcodeLbl);
+    if (initAddressFiller) {
```

Comment:
How about adding `UNLIKELY` keyword here since this if statement is executed only once for initialization?

Reply:
Good idea! (I've updated the patch)

Reply:
@rtakacs Thanks!